package.json

```diff
@@ -20,14 +20,19 @@
     "lint-sass": "sass-lint client/src"
   },
   "dependencies": {
+    "@silverstripe/reactstrap-confirm": "^0.0.4",
     "babel-polyfill": "6.7.4",
+    "mockdate": "^3.0.3",
+    "moment": "^2.24.0",
```

Comment:
Can we use the same constraint that we use in [admin (`^2.22.2`)](https://github.com/silverstripe/silverstripe-admin/blob/036bc9ea97e40d50207d06680366c572f6fb85dc/package.json#L66)? Or do a PR on admin to update to the latest version there.